docs: update readme, remove gemini api key + clean up (#206)
autokitteh/samples: slack_support/README.md

---

> title: AI-Driven Slack Bot for Assistance Requests

Reviewer: ```suggestion
title: AI-driven Slack bot for assistance requests
```

> description: AI-powered Slack bot that automatically routes help requests to the right expert based on topic analysis and expertise matching.

Reviewer: ```suggestion
description: Automatically route help requests to the right expert based on topic analysis and expertise matching
```

> integrations: ["slack", "googlegemini"]

Reviewer: ```suggestion
integrations: ["slack", "sheets", "googlegemini"]
```

---
title: AI-Driven Slack Bot for Assistance Requests
description:

> This Slack bot automates assistance requests using Google's Gemini AI to categorize inquiries. Based on a predefined expertise table stored in a Google Sheet, the bot assigns the request to the appropriate expert. The expert can then use `!take` to claim the request and later mark it as resolved using `!resolve`.

Reviewer: Given that this appears (only) in the repo's table, is it possible to have a much shorter description? Just a high-level teaser.

![demo](./demo.png)

## How It Works

1. Receive a help request from a user
2. Identify the request topic using Gemini AI
3. Assign the request to the appropriate expert based on the expertise table
4. Confirm the expert’s acceptance of the request
5. Track the request until the expert resolves it
6. Remind the expert if the request remains unresolved within a set time

## Cloud Usage

1. Initialize your connections (Google Sheets, Google Gemini, Slack)
2. Set the `DIRECTORY_GOOGLE_SHEET_ID` project variable, in the "VARIABLES" tab, to point to your Google Sheet
3. (Optional) Set the `HELP_REQUEST_TIMEOUT_MINUTES` project variable, in the "VARIABLES" tab, to set the timeout for unclaimed requests
4. Deploy project

## Trigger Workflow

The workflow is triggered when the bot is mentioned in a message within a channel where it is a member. For example:

```
@autokitteh help me with my cat
```

> [!IMPORTANT]

> > For self-hosted, replace `@autokitteh` with your app's name (e.g., `@yourapp help me with my cat`).

Reviewer: This comment isn't relevant only to self-hosted servers, it depends on the Slack app being used in the connection. As such, I'm not sure it's even worth writing, the user ought to know which app they're using.

Author: Fair. Done.
